base/src/com/google/idea/blaze/base/sync/projectview/ImportRoots.java

```diff
@@ -91,6 +92,7 @@ public ImportRoots build() {
       if (buildSystem == BuildSystem.Bazel && hasWorkspaceRoot(rootDirectories)) {
         excludeBuildSystemArtifacts();
       }
+      excludeProjectDataSubDirectory();
```

Comment:
Thanks for the patch, I'll apply this and #497 upstream with some minor modifications (e.g. it's only applicable to bazel projects).

Reply:
👍 great, thank you